src/components/Header/Header.tsx

```diff
@@ -10,6 +10,10 @@ import headerStyles from '../../themes/teams/components/Header/headerStyles'
  * @accessibility
  * Headings communicate the organization of the content on the page. Web browsers, plug-ins, and assistive technologies can use them to provide in-page navigation.
  * Nest headings by their rank (or level). The most important heading has the rank 1 (<h1>), the least important heading rank 6 (<h6>). Headings with an equal or higher rank start a new section, headings with a lower rank start new subsections that are part of the higher ranked section.
+ *
+ *
+ * Other considerations:
+ *  - when description is used in header, then reader narrate both as header text.
```

Comment:
I would change this line to:
when description property is used in header, readers will narrate both header content and description within the element. In addition to that, both will be displayed in the list of headings.